docs/es/README.md

```diff
@@ -0,0 +1,319 @@
+  </figure>
+  <figure>
+    <img src="../de/assets/user-profile.png" alt="Perfil de usuario" />
+    <figcapture>
+      Perfil de usuario
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/user-settings.png" alt="Configuración de la cuenta de usuario" />
+    <figcapture>
+      Configuración de la cuenta de usuario
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/group-list.png" alt="Vista general de grupos" />
+    <figcapture>
+      Vista general de grupos
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/group-profile.png" alt="Perfil del grupo" />
+    <figcapture>
+      Perfil del grupo
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/1-to-1-chat.png" alt="Chat uno a uno" />
+    <figcapture>
+      Chat uno a uno
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/notifications.png" alt="Notificaciones" />
+    <figcapture>
+      Notificaciones
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/map.png" alt="Mapa" />
+    <figcapture>
+      Mapa
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/language-selection.png" alt="Multilingüismo" />
+    <figcapture>
+      Multilingüismo
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/moderation.png" alt="Moderación" />
+    <figcapture>
+      Moderación
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/system-administration.png" alt="Administración del sistema" />
+    <figcapture>
+      Administración del sistema
+    </figcapture>
+  </figure>
+</div>
+
+<div class="all-features">
+  <a id="hero-button" href="/es/features/">
+    <Button>
+      Ver todas las funciones
+    </Button>
+  </a>
+</div>
+
+<!-- XXX translate -->
+
+<!-- <h2 id="why-foss" class="large-header">Why free open source software?</h2>
+
+<div class="benefits-grid center">
+  <figure>
+    <HopeIcon icon="fa-solid" />
+    <figcapture>
+      <h3>Frei verfügbar</h3>
+      Freie Software ist für jeden Menschen jederzeit zugänglich. Dass Lizenzen ablaufen, gibt es nicht.
+    </figcapture>
+  </figure>
+  <figure>
+    <HopeIcon icon="fa-solid" />
+    <figcapture>
+      <h3>Transparent</h3>
+      Bei freier Software ist der Quellcode einsehbar. Jeder Mensch hat so die Möglichkeit nachzusehen, was im Hintergrund eines Programmes passiert.
+    </figcapture>
+  </figure>
+  <figure>
+    <HopeIcon icon="fa-solid" />
+    <figcapture>
+      <h3>Kopierbar</h3>
+      Ich kann jederzeit mit allen meinen Kolleginnen und Kollegen zusammenarbeiten. Alle können es jederzeit verwenden, ohne Einschränkungen.
+    </figcapture>
+  </figure>
+  <figure>
+    <HopeIcon icon="fa-solid" />
+    <figcapture>
+      <h3>Veränderbar</h3>
+      Jede Person kann etwas zu ihrem Lieblingsprogramm beitragen, jederzeit. Ich kann das Programm so verwenden, wie ich es am liebsten will.
+    </figcapture>
+  </figure>
+</div> -->
```

Comment:
Please remove this leftver fr omcopying.

Reply:
To get the Spanish version onto the website now I allow my self to take this as a note in the follow up issue #158 .